docs/docs/reference/production-deployment.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Production Deployment
+description: >-
+  This article covers production deployment requirements and concerns
+---
+
+# Production Deployment
+
+This page covers the topic of running Pomerium in a production configuration.  See the [quick start section](../quick-start/) for canned example configurations.  
+
+Please also see [architecture](../#architecture) for information on component interactions.
+
+## Service Mode
+
+### All in One
+
+In smaller deployments or while testing, it may be desirable to run in "all-in-one" mode.  This reduces resource footprint and simplifies DNS configuration.  All URLs point at the same pomerium service instance.
+
```

Comment:
Should we give the setting?
> All in one mode can be used by setting the services setting to "all

Or something like that ?

Reply:
Linked to the config reference at the top.  Does that make sense?

Reply:
I see links to quickstart and architecture but not reference.

Reply:
Sorry - at the top of the Service Mode section

Reply:
Got it. LGTM